express-zod-api/src/routing.ts

```diff
@@ -81,6 +81,8 @@ export const initRouting = ({
   const deprioritized = new Map<string, RequestHandler>();
   for (const [path, methods] of familiar) {
     const accessMethods = Array.from(methods.keys());
+    /** @link https://github.com/RobinTail/express-zod-api/discussions/2791#discussioncomment-13745912 */
+    if (accessMethods.includes("get")) accessMethods.push("head");
```

Comment:
should this be configurable?

Reply:
probably not, because it depicts the actual behavior of Express regarding `app.get()` method